### ISSUE TITLE
Incompatibility with pyVHDLModel>=v0.10.0

### ISSUE THREAD

**Issue by lwaeijen** [Paebbels/pyVHDLParser#24]
On a fresh install the latest pyVHDLModel package is pulled in (v0.11.1 at the time of writing), which lacks the LibraryReference class. Trying to invoke the cli results in the following error:
```python
from pyVHDLModel.VHDLModel import LibraryReference as LibraryReferenceModel, PackageReference as UseModel
ImportError: cannot import name 'LibraryReference' from 'pyVHDLModel.VHDLModel'
```
It appears this particular class was removed in v0.10.0 of pyVHDLModel, so perhaps the requirements.txt file should be updated to reflect this dependency.

**Comment by Paebbels** [comment]
The `dev` branch now contains a compatible version to v0.14.x.

**Comment by Paebbels** [comment]
This will be addressed in #48.

**Comment by Paebbels** [comment]
Unit tests on CI are passing.
